rero_ils/modules/loans/api.py

```diff
@@ -305,3 +306,19 @@ def get_item_on_loan_loans():
         loan = Loan.get_record_by_pid(record.pid)
         item_on_loan_loans.append(loan)
     return item_on_loan_loans
+
+
+def loan_build_org_ref(data):
+    """Build $ref for the organisation of the Loan."""
+    from ..items.api import Item
+    item_pid = data.get('item_pid')
+    org_pid = Item.get_record_by_pid(item_pid).organisation_pid
+    base_url = current_app.config.get('RERO_ILS_APP_BASE_URL')
+    url_api = '{base_url}/api/{doc_type}/{pid}'
+    org_ref = {
+        '$ref': url_api.format(
+            base_url=base_url,
+            doc_type='organisations',
+            pid=org_pid)
+    }
+    data['organisation'] = org_ref
```

Comment:
For me, this function should be a private function of `Loan` class (def _loan_build_org_ref(self, data): ? ).
I don't think any other function/fixture used this function outside of `Loan` class scope.
(Just like the function "loan_build_item_ref" (line 92)